monkey/agent_plugins/build.sh

```diff
@@ -29,4 +29,4 @@ docker run \
     --rm \
     -v "$SCRIPT_DIR:/plugins" \
     infectionmonkey/agent-builder:$TAG \
-    /bin/bash -c "$DOCKER_COMMANDS" | ts  '[%Y-%m-%d %H:%M:%S]'
+    /bin/bash -ci "$DOCKER_COMMANDS" | ts  '[%Y-%m-%d %H:%M:%S]'
```

Comment:
From the bash manpage: "If the -i option is present, the shell is interactive."

Why would we want an interactive shell? Isn't this script supposed to automate things for us?

Reply:
The agent-builder image was updated to include the rust compiler. I believe the rust compiler is installed per-user, so the rust compiler was not being found in the PATH.  Using `-i` causes bash to load the user config which will update the PATH variable. Alternatively, we might be able to `source` that file in the DOCKER_COMMANDS

Reply:
I went with setting `PATH` as that won't break things if the rust compiler gets removed